### PR TITLE
Dev/document jio 3.6.3

### PR DESCRIPTION
**Pull request by lav-mudgal-guavus**

### CODE REVIEW

File: core-plugins/docs/Validator-transform.md [hunk 1]
```diff
@@ -1,63 +1,87 @@
 # Validator Transform
 
-
 Description
 -----------
-Validates a record, writing to an error dataset if the record is invalid.
```
Comment (deepak-maheshwari-ext) [inline core-plugins/docs/Validator-transform.md:6]: please rollback this deleted table

File: core-plugins/docs/Validator-transform.md [hunk 2]
```diff
@@ -1,63 +1,87 @@
...
 
-Checks if the value's length is greater than or equal to the min.
-Min value should be the minimum length with which validation performs on column value.
-Returns True if the column value's length is more than the specified minimum value. 
+Checks if the length of the value in the input column is more than or equal to the specified minimum length. The function takes a column name and the allowed minimum length as inputs.
+
+Returns `True` if the column value's length is more than or equal to the specified minimum length. 
 
 Use Case
 --------
```
Comment (deepak-maheshwari-ext) [inline core-plugins/docs/Validator-transform.md:60]: please rollback this use case

**Review by deepak-maheshwari-ext — CHANGES_REQUESTED** [review]

File: core-plugins/docs/File-batchsource.md [hunk 3]
```diff
@@ -2,46 +2,44 @@
 
 
 ## Description
+File Batch Source is a Guavus Enterprise Accelerator that is used when you want to read from a distributed file system.
 
-This source is used whenever you need to read from a distributed file system.
-For example, you may want to read in log files from S3 every hour and then store
-the logs in a TimePartitionedFileSet.
+## Use Case
+Consider a scenario wherein you want to fetch log files from S3 every hour and then store the logs in a TimePartitionedFileSet. It can be achieved my making configurational changes as described in the sections below.
```
Comment (deepak-maheshwari-ext) [inline core-plugins/docs/File-batchsource.md]: change s3 to hdfs in every place of doc . means "fetch file from hdfs".

File: core-plugins/docs/File-batchsource.md [hunk 4]
```diff
@@ -2,46 +2,44 @@
...
 
+## Configuration
+The following pointers describe the fields as displayed in the accelerator properties dialog box.
 
-## Properties
+**Reference Name:** The name used to uniquely identify this source for lineage, annotating metadata, etc.
 
-**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+**Path:** The path to read from. For example, s3a://<bucket>/path/to/input
```
Comment (deepak-maheshwari-ext) [inline core-plugins/docs/File-batchsource.md]: change path like that . "hdfs://<host>/tmp/sample.txt"

**Review by deepak-maheshwari-ext — CHANGES_REQUESTED** [review]

File: core-plugins/docs/Validator-transform.md [hunk 5]
```diff
@@ -1,63 +1,132 @@
...
+| isValidCountryCodeTid(String ccTld)             | Returns true if the input matches any IANA-defined             |
+|                                                 | country code top-level domain                                  |
+| isValidGenericTId(String gTld)                  | Returns true if the input matches any IANA-defined             |
+|                                                 | generic top-level domain                                       |
+| isValidInfrastructureTId(String iTld)           | Returns true if the input matches any IANA-defined             |
+|                                                 | infrastructure top-level domain                                |
+| isValidLocalTId(String lTld)                    | Returns true if the input matches any widely used              |
+|                                                 | local domains (localhost or localdomain)                       |
+| isValidTId(String tld)                          | Returns true if the input matches any IANA-defined             |
+|                                                 | top-level domain                                               |
```
Comment (deepak-maheshwari-ext) [inline core-plugins/docs/Validator-transform.md]: need to add only deleted table. this table is invalid

**Review by deepak-maheshwari-ext — CHANGES_REQUESTED** [review]

**Comment by lav-mudgal-guavus** [comment]
merged in #137

**Pull request closed without merging**